MediaGalleryUi/view/adminhtml/web/js/image-uploader.js

```diff
@@ -112,6 +114,9 @@ define([
 
                         return;
                     }
+                    this.sortBy().selectDefaultOption();
```

Comment:
In this case if we upload for example 10 images, this operation will  be executed 10 time's

Reply:
I would recommend to move this operation to stop method to run this on last uploaded image